bin/users.json

```diff
@@ -69,7 +69,7 @@
 
   {"email": "ts.service.translation-manage@gmail.com", "roles": "ccd-import,manage-translations", "lastName": "Translations", "firstName": "Manage"},
   {"email": "ts.service.translation-load@gmail.com", "roles": "ccd-import,load-translations", "lastName":"Translations", "firstName": "Load"},
-  {"email": "ts.service.translation-manage-load@gmail.com", "roles": "ccd-import,load-translations,manage-translations", "lastName": "Translations", "firstName": "Manage-Load"}
-
+  {"email": "ts.service.translation-manage-load@gmail.com", "roles": "ccd-import,load-translations,manage-translations", "lastName": "Translations", "firstName": "Manage-Load"},
 
+  {"email": "next.hearing.date.admin@gmail.com", "roles": "caseworker,caseworker-befta_master,next-hearing-date-admin", "lastName": "hearing", "firstName": "next-day"}
```

Comment:
The `next.hearing.date.admin` account should not have jurisdiction roles assigned. It should get access to the case by having the `next-hearing-date-admin` role which should be configured in data-store as one of the `cross-jurisdictional-roles`.

Reply:
I remember that we mention about some changes in data-store to have a predefined supper role like caseworker-caa .. but can you suggest me exactly the changes should be please ?    "roles": "caseworker,caseworker-befta_master,next-hearing-date-admin"  
}

Reply:
data store change has been raised in the following PR

https://github.com/hmcts/ccd-data-store-api/pull/2078

Reply:
This has been updated 👍